packages/cli/src/commands/exec/simulateFileAccess.ts

```diff
@@ -7,20 +7,40 @@ export async function simulateFileAccess(logger: Logger, inputs: string[], outpu
   const root = getWorkspaceRoot(process.cwd())!;
   logger.silly("Now probing and touching inputs and outputs");
 
+  const inputDirectories = new Set<string>();
+
   // probe input files
   let fd: number;
   for (const input of inputs) {
     fd = fs.openSync(path.join(root, input), "r");
     fs.closeSync(fd);
+
+    inputDirectories.add(path.dirname(input));
+  }
+
+  for (const directory of inputDirectories) {
+    fd = fs.openSync(path.join(root, directory), "r");
+    fs.closeSync(fd);
   }
 
```

Comment:
@kenotron , could you double check that this is actually interpreted as an enumeration? If you run a build with a lage client using this in debug mode, bxl should be reporting an enumeration on the given directory. That's an indication that this is actually doing what we want